src/arrayMethodSort.js

```diff
@@ -3,9 +3,40 @@
 /**
  * Implement method Sort
  */
+
+const compareAsStrings = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringA === stringB) {
+    return 0;
+  } else {
+    return -1;
+  }
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareAsStrings) {
```

Comment:
Good! but you can make the same shorter
```suggestion
  [].__proto__.sort2 = function(compareFunction = (a, b) => String(a) > String(b)) {
```